pkg/envoy/sds/response.go

```diff
@@ -37,7 +38,7 @@ func NewResponse(meshCatalog catalog.MeshCataloger, proxy *envoy.Proxy, request
 	log.Info().Str("proxy", proxy.String()).Msgf("Creating SDS response for request for resources %v", requestedCerts)
 
 	// 1. Issue a service certificate for this proxy
-	cert, err := certManager.IssueCertificate(certificate.CommonName(s.serviceIdentity), cfg.GetServiceCertValidityPeriod())
+	cert, err := certManager.IssueCertificate(s.serviceIdentity.String())
```

Comment:
Is there a reason we're not setting a validity duration on this cert? Do we already know the cert exists which basically turns this into a cache lookup and the cert will already have one defined?

Reply:
The manager has a default validity period it uses, which we're getting from the MRC, so won't have access through the configurator in the future.

Removed configurator from the sdsimpl